Packs/Unit42ThreatIntelligencebyPaloAlto/Integrations/Unit42Feed/Unit42Feed.yml

```diff
@@ -0,0 +1,179 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Unit 42 Feed
+  version: -1
+sectionOrder:
+- Connect
+- Collect
+configuration:
+- display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
+  defaultvalue: 'true'
+  section: Collect
+- display: Feed Types
```

Comment:
@MosheEichler, please add tags as an integration parameter as well (ability to add tags to fetched indicators - should be standard)